x-pack/plugins/beats_management/public/components/tag/config_view/config_form.tsx

```diff
@@ -15,8 +15,23 @@ import { FormsyEuiFieldText } from '../../inputs/input';
 import { FormsyEuiMultiFieldText } from '../../inputs/multi_input';
 import { FormsyEuiSelect } from '../../inputs/select';
 
-addValidationRule('isHost', (values: FormData, value: FieldValue) => {
-  return value && value.length > 0;
+addValidationRule('isHosts', (form: FormData, values: FieldValue | string[]) => {
+  if (values && values.length > 0 && values instanceof Array) {
+    return values.reduce((pass: boolean, value: string) => {
+      if (
+        value.match(
```

Comment:
Can we consider refactoring this check into a function so that `prettier` doesn't have an aneurism trying to make it readable? 😆